web/src/components/Navigation/useNavigation.ts

```diff
@@ -10,9 +13,9 @@ import { useSession } from "src/auth";
 // the viewport. To get around this, the default layout positions an empty box
 // to the left of the viewport to push the content right and then the actual
 // sidebar is rendered on top of this with the same width configuration.
-export const SIDEBAR_WIDTH = {
-  md: "25%",
-  lg: "33%",
+export const SIDEBAR_WIDTH: PropertyValue<"width"> = {
```

Comment:
Here I've switched this to use Panda's tokens instead (which are nicer)

Though, as the aforementioned comment mentions, this doesn't work as a JS variable.